src/test/java/nextstep/ladder/domain/ladder/LadderGameTest.java

```diff
@@ -0,0 +1,48 @@
+package nextstep.ladder.domain.ladder;
+
+import nextstep.ladder.domain.strategy.FixedLadderLineGenerateStrategy;
+import nextstep.ladder.domain.strategy.RandomLadderLineGenerateStrategy;
+import nextstep.ladder.domain.user.User;
+import nextstep.ladder.domain.user.Users;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+class LadderGameTest {
+
+    private static final Users U1 = new Users("pobi,honux,crong,jk");
+    private static Lines L1;
+    private static Rewards R1 = new Rewards("꽝,3000,꽝,5000");
+
+    @BeforeEach
+    void setup() {
+        L1 = new Lines(
+                new Line(4, new FixedLadderLineGenerateStrategy(true, false, true)),
+                new Line(4, new FixedLadderLineGenerateStrategy(false, true, false)),
+                new Line(4, new FixedLadderLineGenerateStrategy(true, false, false)),
+                new Line(4, new FixedLadderLineGenerateStrategy(false, false, true))
+        );
```

Comment:
```suggestion
Lines L1 = new Lines(
    new Line(4, size -> List.of(true, false, true)),
    new Line(4, size -> List.of(false, true, false)),
    new Line(4, size -> List.of(true, false, false)),
    new Line(4, size -> List.of(false, false, true))
);
```
위와 같이 람다로 구현해도 되지 않을까?